inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/lrn/lrn_kernel_across_channel_multiple_features.cpp

```diff
@@ -67,11 +67,11 @@ CommonDispatchData LRNKernelAcrossChannelMultipleFeatures::SetDefault(const lrn_
     if (input.GetLayout() == DataLayout::bfyx || input.GetLayout() == DataLayout::b_fs_yx_fsv4 ||
         input.GetLayout() == DataLayout::b_fs_yx_fsv16) {
         const auto& out = params.output;
-        const unsigned int alignment = out.X().v > 16 ? 32 : 16;
+        const unsigned int alignment = out.Feature().v > 16 ? 32 : 16;
 
-        runInfo.gws0 = Align(out.X().v, alignment);
+        runInfo.gws0 = Align(out.Feature().v / ofm_per_simd, alignment);
```

Comment:
This will most likely cause a regression if lrn is used with bfyx format. Wouldn't it be better to have two modes - simd along x (bfyx) and simd along f (b_fs_yx_fsv16)?

Reply:
Yes, you are right. Thank you!